src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/MinMax.txt

```diff
@@ -0,0 +1,50 @@
+﻿>> Max(1,2)
+2
+
+>> Max(Blank(), 100, -5, "2000", Value("2001"), Text(2002))
+2002
+
+>> Max("200", "2000")
+2000
+
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 11, 2014 1:50 PM"))
+11/11/2014 01:50:00 PM
+
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 10, 2014 2:50 PM"))
+11/10/2014 02:50:00 PM
+
+>> Max(Time(1,50,0),Time(2,0,0))
+02:00:00
+
+>> Min(1,2)
+1
+
+>> Min(Date(1985,3,5),Date(1986,10,19))
+Date(1985,3,5)
+
+>> Min(-100, If(true, -1000, 200))
+-1000
+
+>> Min([1, 2, 3, 4, 5], Value)
+1
+
+>> Min([1, 2, 1/0, 4, 5], Value)
+#Error
+
+>> Min(1, 2, 3, Blank(), 2, 3)
+1
+
+>> Max([1, 2, 3, 4, 5], Value)
+5
+
+>> Max([1, 2, 1/0, 4, 5], Value)
+#Error
+
+>> Max(1, 2, 3, Blank(), 2, 3)
+3
+
+>> Max(Table({Name:"John Doe", DOB:Date(1985,3,5)},{Name:"Jane Roe", DOB:Date(1986,10,19)}),DOB)
+Date(1986,10,19)
+
+>> Max(Table({Name:"John Doe", DOB:Date(1985,3,5), Thing: "1"},{Name:"Jane Roe", DOB:Date(1986,10,19), Thing: "2"}),Thing)
+2
```

Comment:
More tests:

- Single arguments (this is valid): `Max(1)`, `Min(Date(2022,3,8))`, ...
- Min/max of very large tables: `Max(Sequence(33333, 10) => 33342`, `Min(Sequence(10000), DateAdd(Date(2000,1,1), -Value, TimeUnit.Days) => Date(1992,8,5)`, ...
- Errors in multi-parameter overloads: `Min(1, 3, Sqrt(-1), -3) => #Error(Kind=Numeric)`, `Max(Time(3,4,5), Time(2,1/0,8)) => #Error(Kind=Div0)`, ...

Reply:
Added these. Thanks!